<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>영역</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_1">
     <property name="text">
      <string>기초교양</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>인성교양</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>일반선택영역</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>자연과학영역</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>사회과학영역</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>인문예술영역</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QCheckBox" name="checkBox_7">
     <property name="text">
      <string>디지털 리터러시영역</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="checkBox_8">
     <property name="text">
      <string>교직영역</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="9">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>삼육대학교 공통교양 조회</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="9">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>과목코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>과목명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학년</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>이수구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>영역구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>학점</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>교수명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>수업시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>장소</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>비고</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
